cmd/byohctl/cmd/onboard.go

```diff
@@ -35,28 +35,24 @@ This command will:
 1. Authenticate with Platform9
 2. Get required configuration
 3. Setup the host for management`,
-	Example: `  byohctl onboard -u admin@platform9.com -p password -f your-fqdn.platform9.com
-  byohctl onboard -u admin@platform9.com -p password -f your-fqdn.platform9.com -d custom-domain -t custom-tenant`,
+	Example: `  byohctl onboard -u your-fqdn.platform9.com -e admin@platform9.com -c client-token
```

Comment:
The help text says both `u` and `e` flags are for username : if both are provided, what takes precedence? 
Also, I don't think we are doing any validation for the format like is it an email, is it a valid FQDN...

```
	onboardCmd.Flags().StringVarP(&username, "username", "u", "", "Platform9 username")
	onboardCmd.Flags().StringVarP(&fqdn, "url", "u", "", "Platform9 FQDN")
	onboardCmd.MarkFlagRequired("url")
	onboardCmd.Flags().StringVarP(&username, "username", "e", "", "Platform9 username")
```

Reply:
Thanks for updating it.